R/include-gld-purchases.R

```diff
@@ -0,0 +1,33 @@
+#' Include only those who have a purchase of a glucose lowering drug (GLD).
+#'
+#' See [algorithm] for the logic used to filter these patients.
+#'
+#' @return The same type as the input data, default as a [tibble::tibble()].
+#' @keywords internal
+#'
+#' @examples
+#' \dontrun{
+#' register_data$lmdb |> include_gld_purchases()
+#' }
+include_gld_purchases <- function(data) {
+  verify_required_variables(data, "lmdb")
```

Comment:
```suggestion
include_gld_purchases <- function(lmdb) {
  verify_required_variables(lmdb, "lmdb")
```
At least according to #142